sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -513,7 +518,7 @@ def get_tx_fault(self):
         return tx_fault
 
     def get_tx_los_support(self):
-        return not self.is_flat_memory()
+        return not self.is_flat_memory() and self.xcvr_eeprom.read(consts.TX_LOS_SUPPORT_FIELD)
```

Comment:
this field is inside reg group MODULE_CHAR_ADVT_FIELD so cannot be read directly.